src/mjolnir/CMakeLists.txt

```diff
@@ -43,6 +43,7 @@ set(sources
   hierarchybuilder.cc
   ingest_transit.cc
   landmarks.cc
+  traffic_updater.cc
```

Comment:
IMO it makes sense to rename this to something like `add_predicted_speeds.cc`, because traffic_updater is a little broad and Valhalla also supports live traffic for example.